ArduPlane/altitude.cpp

```diff
@@ -289,12 +289,11 @@ void Plane::change_target_altitude(int32_t change_cm)
 {
     target_altitude.amsl_cm += change_cm;
 #if AP_TERRAIN_AVAILABLE
-    if (target_altitude.terrain_following) {
+    if (target_altitude.terrain_following && !nonauto_terrain_disable ) {
```

Comment:
I don't think this one should change

Reply:
changed it anyway....doesn't hurt does it?also realized that I needed to set target alt on switch changes also...it would not shift modes until you touched the elev stick in CRUISE/FBWB...was thinking it would be okay, but decided better to have it change to/from terrain following transparently...and figured how to do it in RC_Channel...but its probably not correct style...works fine however....